jstp/src/main/java/com/metarhia/jstp/JSTP.java

```diff
@@ -7,17 +7,32 @@
 
 /**
  * Common JSTP interface
+ *
  * Created by lidaamber on 18.06.16.
  */
 public final class JSTP implements Serializable {
 
   private JSTP() {
   }
 
+  /**
+   * Parses string representation of JS value
+   *
+   * @param input string representation of JS value
+   *
+   * @return parsed JS value
+   */
   public static <T> T parse(String input) throws JSParsingException {
     return JSParser.parse(input);
   }
 
+  /**
+   * Creates string representation of JS value
+   *
+   * @param value parsed JS value
+   *
+   * @return string representation of JS value
+   */
   public static <T> String stringify(T value) {
```

Comment:
Also, we should put that description to the JSSerializer.stringify and just have a link here.